packages/mux-video-ads/src/index.ts

```diff
@@ -23,6 +23,7 @@ const Attributes = {
 
 class MuxVideoAds extends MuxVideoElement {
   #muxAdManager: MuxAdManager | undefined;
+  #isUsingSameVideoElement: boolean | undefined;
```

Comment:
suggestion: just make this a private getter so we don't have to copy the value or worry about the values getting out of sync, e.g.
`get #isUsingSameVideoElement () { return this.#muxAdManager?.isUsingSameVideoElement(); }`